data/products.yml

```diff
@@ -7,6 +7,10 @@ influxdb3_core:
   list_order: 2
   latest: core
   placeholder_host: localhost:8181
+  sample_questions:
```

Comment:
I'm wondering if we should call this `ai_sample_questions`, just to provide more specific context.